### ISSUE TITLE
Add best in class logo to site

### ISSUE THREAD

**Issue by vickimcfadden** [GSA/oasis-plus#381]

**Comment by PowderD** [comment]
![pasted image 0 (1)](https://github.com/GSA/oasis-plus/assets/147180417/e3883e75-bb20-4ad0-ab3d-eb6077743ec6)

Let me know if this works or if it's too small.  @weiwang-gsa @vickimcfadden

**Comment by klohman** [comment]
# Proposed design: 

## About OASIS+ page 
- Recommend adding to the about page, where BIC is mentioned in context 
![image](https://github.com/GSA/oasis-plus/assets/18366192/e15c8067-96c3-4f55-aaaf-3f1c418eae3d)

We can also consider adding it to the homepage, although Best-In-Class is not specifically mentioned there, if there needs to be more attention drawn to this upfront. @ekidenda @PowderD

**Comment by PowderD** [comment]
@klohman @vickimcfadden @ekidenda I think it needs to be on the homepage.  It is a big thing to have BIC designation and it's important for us to highlight it, whether that's just the logo or mentioning it on the homepage as well.  It looks good on the about page.  Thank you!

**Comment by klohman** [comment]
@PowderD @ekidenda 

Here are some options for your consideration for adding the BIC info in the homepage

### Option 1 - Add to banner as 4th "star" (this option doesn't use the BIC logo)
![Image](https://github.com/GSA/oasis-plus/assets/18366192/dd72d6a1-a8b3-45f3-ba91-2f4fc3ede28c)

### Option 2 - Add to banner and replace "multiple award IDIQ" (this option doesn't use the BIC logo)
![Image](https://github.com/GSA/oasis-plus/assets/18366192/eadd98f7-bfa0-4744-b0f6-8ed5bd985c78)

### Option 3 - Add BIC section below role-based cards 
![Image](https://github.com/GSA/oasis-plus/assets/18366192/a0cfb5d7-352b-4d9b-a53f-982319b79af4)


cc: @rosamundtgov

**Comment by PowderD** [comment]
@klohman Could we do a combo of Option 1 and Option 3?  I like having it at the top in the blue/white, but also like the logo as the BIC team is really fond of their logo being attached.  Could we move that section above "Buying Through OASIS"?  @ekidenda Any other thoughts or if you prefer something different please feel free to share.  

Thank you for starting this. Looking great!

**Comment by ekidenda** [comment]
I like the idea you have @PowderD as I was also thinking along the lines of moving the logo up on the page. Combining options 1 and 3 would be ideal.

**Comment by klohman** [comment]
@ekidenda @PowderD I'd suggest not moving the BIC section (option 3) above the task-based cards, since those are the reasons that users are coming to the site. The addition in the banner and then the supplemental information further down the homepage, and prominently on the About OASIS+ page should be able to inform users of the designation without interrupting or blocking them from getting to their core content. @rosamundtgov Looking for your thoughts?

**Comment by rosamundtgov** [comment]
agreed on not blocking the users from the information they came here for. @klohman let's see if there are any options that can incorporate the logo but not require more scrolling. 

> @ekidenda @PowderD I'd suggest not moving the BIC section (option 3) above the task-based cards, since those are the reasons that users are coming to the site. The addition in the banner and then the supplemental information further down the homepage, and prominently on the About OASIS+ page should be able to inform users of the designation without interrupting or blocking them from getting to their core content. @rosamundtgov Looking for your thoughts?

**Comment by vickimcfadden** [comment]
@rosamundtgov to confirm Best-In-Class or Best-in-class or Best-in-Class

**Comment by weiwang-gsa** [comment]
@rosamundtgov please let me know what is the final text for it, thank you.

**Comment by weiwang-gsa** [comment]
@klohman this is ready for design review at feature branch: https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/wwang/best-in-class/

**Comment by klohman** [comment]
@weiwang-gsa Homepage update looks good.

**Comment by weiwang-gsa** [comment]
@ekidenda or @PowderD this is ready for QA review here:https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/demo/gsa/oasis-plus/
